src/components/sections/courses/overview/index.tsx

```diff
@@ -23,15 +23,15 @@ export default function Overview(): ReactElement {
   if (!course) return <></>;
   return (
     <div className="flex flex-col divide-y divide-solid divide-gray-200 lg:py-5 text-gray-700">
-      <Header title={course.name} description={course.description} />
-      <RewardsSection />
-      <ObjectivesSection />
-      <PrerequisiteSection />
-      <DisclaimerSection />
-      <TrailerSection />
-      <LearningModulesSection />
-      <ChallengeSection />
-      <PageNavigation />
+      <Header data-testid="headerId" title={course.name} description={course.description} />
+      <RewardsSection data-testid=""/>
+      <ObjectivesSection data-testid=""/>
+      <PrerequisiteSection data-testid=""/>
+      <DisclaimerSection data-testid="disclaimerSectionId"/>
+      <TrailerSection data-testid="trailerSectionId"/>
+      <LearningModulesSection data-testid="learningModulesSection" />
+      <ChallengeSection data-testid="challengeSectionId"/>
+      <PageNavigation data-testid="pageNavigationId"/>
     </div>
```

Comment:
let us avoid this kind of implementation of passing `data-testid` on every imported component to test if it is available in the current component